src/Powercord/plugins/pc-routelinks/index.js

```diff
@@ -0,0 +1,90 @@
+const { Plugin } = require('powercord/entities');
+const { getRepoInfo, cloneRepo } = require('../pc-moduleManager/util');
+const express = require('express');
+const { open: openModal, close: closeModal } = require('powercord/modal');
+const { React } = require('powercord/webpack');
+const Modal = require('./components/ConfirmModal');
+
+
+module.exports = class RDLinks extends Plugin {
+  async startPlugin () {
+    this.app = express();
+
+    this.app.use((req, res, next) => {
+      res.header('Access-Control-Allow-Origin', 'https://replugged.dev');
+      res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+      res.header('Access-Control-Allow-Methods', 'POST');
+      next();
+    });
+
+    this.app.post('/install/', async (req, res) => {
+      this.info = await getRepoInfo(req.query.address);
+      if (this.info) {
+        if (this.info.isInstalled) {
+          res.status(403).json({ error: 'ALREADY-INSTALLED',
+            plainText: `${this.info.repoName} is already installed.`
+          });
+          return;
+        }
+        // eslint-disable-next-line no-warning-comments
+        // TODO QUEUE
+        res.status(200).json({ error: 'SUCCESS',
+          plainText: `Successfully sent prompt for ${this.info.repoName} install.`
+        });
+        this.info.url = req.query.address;
+        this.openInstallModal();
+      } else {
+        res.status(404).json({ error: 'CANNOT-FIND',
+          plainText: `Cannot find ${req.query.address}.`
+        });
+      }
+    });
+
+
+    this.httpserv = this.app.listen(6473);
+  }
+
+  openInstallModal () {
+    global.DiscordNative.window.focus();
+
+    openModal(() => React.createElement(Modal, {
+      red: true,
+      header: `Install ${this.info.type}`,
+      desc: `Are you sure you want to install the ${this.info.type} ${this.info.repoName} from ?`,
+      url: this.info.url,
```

Comment:
Some formatting to fix here
<img width="449" alt="Screenshot on 2022-08-07 at 19 19 45" src="https://user-images.githubusercontent.com/14863373/183317026-af40bcf9-f3e2-4e45-86e6-ca6e4ef7b2e8.png">

Reply:
I think the `$` is from below